packages/design-tokens/docs/DocsComponents.tsx

```diff
@@ -70,7 +70,6 @@ const TabbedCodeBlocks = ({
               // eslint-disable-next-line jsx-a11y/anchor-is-valid
               <a
                 style={{ flexShrink: 0 }}
-                data-automation-id={tab.automationId}
```

Comment:
Do we need to remove the `automationId` prop then? Or put it into a `data-testid`?

Reply:
Definitely probably don't need this